src/Symfony/Component/Translation/Loader/XliffFileLoader.php

```diff
@@ -148,7 +148,7 @@ private function parseFile($file)
         $source = str_replace('http://www.w3.org/2001/xml.xsd', $location, $source);
 
         if (!@$dom->schemaValidateSource($source)) {
-            throw new InvalidResourceException(implode("\n", $this->getXmlErrors($internalErrors)));
+            throw new InvalidResourceException(sprintf('The "%s" is not a valid resource: ', $file, implode("\n", $this->getXmlErrors($internalErrors))));
```

Comment:
`The "my_file.xml" is not a valid resource`, This sounds a bit weird. What if you format it like this? 

`'Invalid resource provided: "%s"; Errors: %s'`, note that you forgot the second `%s` in your commit.